pkg/monitoring/alloy/templates/alloy-config.alloy.template

```diff
@@ -35,35 +43,74 @@ prometheus.operator.podmonitors "default" {
   }
 }
 
-prometheus.remote_write "default" {
+// we create a podmonitor and servicemonitor component per tenant because we cannot read pod/service monitor labels through relabelling.
+{{ range .Tenants }}
+prometheus.operator.servicemonitors "{{ . }}" {
+  forward_to = [prometheus.remote_write.{{ . }}.receiver]
+  selector {
+    match_expression {
+      key      = "observability.giantswarm.io/tenant"
+      operator = "In"
+      values   = [{{ . | quote }}]
```

Comment:
Feels strange to use `quote` here but not in other places, either remove it here or add it in other places.

Reply:
fixed